main.py

```diff
@@ -24,6 +24,12 @@ def get_args():
         help="Experimental! Generate RDF in turtle format",
     )
 
+    argparser.add_argument(
+        "--gen-shacl",
```

Comment:
I don't think a new option is needed; we still generate RDF turtle -- SHACL is the way to express the restrictions in this format.